core/store/src/trie/shard_tries.rs

```diff
@@ -90,6 +92,10 @@ impl ShardTries {
         TrieUpdate::new(self.get_view_trie_for_shard(shard_uid, state_root))
     }
 
+    pub fn get_shard_uids(&self) -> &[ShardUId] {
```

Comment:
It's used by the runtime params estimator, in `estimator::apply_action_receipt`.
The `EstimatorContext::testbed`  function manually creates data for a shard, there's no epoch manager or shard layout.
It would be possible to refactor the estimator to use a shard layout, but the easiest way was to get the list of shards from the list of tries per shard. AFAIU there's only one shard at the moment, so it's not a big issue.